src/flows/shared/steps/OrganizeCollection/Sidebar/Sidebar.tsx

```diff
@@ -152,7 +152,7 @@ const Header = styled.div`
   display: flex;
   justify-content: space-between;
   align-items: baseline;
-  min-height: 52px;
+  min-height: calc(52px - 16px); /* Matches the padding of StyledSidebar */
```

Comment:
would it be simpler to add padding-bottom: 16px? (or equivalent <Spacer/>)

<img width="589" alt="Screen Shot 2022-07-06 at 16 32 26" src="https://user-images.githubusercontent.com/80802871/177588341-5005ce04-323b-4f21-bd07-158ab203883a.png">